recipe/meta.yaml

```diff
@@ -32,17 +32,26 @@ build:
 # documentation is not built.
 requirements:
     build:
-        - cmake >=2.8.6
+        - cmake >=3.1.3
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
+        - xorg-libxi              # [linux]
+        - xorg-libxmu             # [linux]
+        - libglu                  # [linux]
```

Comment:
These were install by yum previously, can probably remove that yum file now. I think it is deprecated.

Reply:
Okay. I can't get Simbody to find GL.h either way yet.